### ISSUE TITLE
No image alternative text for Alfresco logo

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#375]
To comply to WCAG 2.0 AA standards the Alfresco logo should have alternative text
![screen shot 2016-07-06 at 15 03 13](https://cloud.githubusercontent.com/assets/13200338/16620571/e0580868-438a-11e6-9d6a-03ef19f7da75.png)

**Comment by VitoAlbano** [comment]
Fixed.
![screen shot 2016-07-11 at 18 12 48](https://cloud.githubusercontent.com/assets/7974125/16739655/20b1552a-4793-11e6-87bd-b18025a0d8b4.png)
